modules/eks/variables.tf

```diff
@@ -45,8 +45,9 @@ variable "iam_account_id" {
 
 variable "iam_role_mapping" {
   type = list(object({
-    name = string
-    arn  = string
+    name   = string
+    arn    = string
+    groups = list(string)
   }))
   description = "List of mappings of users to roles"
```

Comment:
Maybe let's change this to "List of mappings of AWS Roles to Kubernetes Groups" ?

Reply:
good done